src/styles/overwrite-tachyons.scss

```diff
@@ -34,396 +34,20 @@ code,
   outline-offset: -1px;
 }
 
-// Colors
+// Colors: Iterate over all variables to create .color, .bg-color and .b--color clases at the same time
 
-// Blue
+$colors: blue-50, blue-60, blue-70, blue-80, blue-90, teal-50, teal-60, teal-70, teal-80, teal-90, magenta-50, magenta-60, magenta-70, magenta-80, magenta-90, green-50, green-60, green-70, green-80, green-90, yellow-50, yellow-60, yellow-70, yellow-80, yellow-90, red-50, red-60, red-70, red-80, red-90, purple-50, purple-60, purple-70, purple-80, purple-90, ink-70, ink-80, ink-90, grey-10, grey-30, grey-50, grey-70, grey-90;
```

Comment:
Is this possible even when colors like blue and ink for example do not have the same amount of shades? 5 vs 3.